radix-engine/tests/metering.rs

```diff
@@ -0,0 +1,40 @@
+#[rustfmt::skip]
+pub mod test_runner;
+
+use radix_engine::ledger::*;
+use radix_engine::transaction::*;
+use radix_engine::wasm::InvokeError;
+use scrypto::call_data;
+use scrypto::prelude::*;
+
```

Comment:
Can we also add max_stack_depth test and memory_grow test?

Reply:
Yes, will add